charts/posthog/README.md

```diff
@@ -16,47 +16,129 @@ This chart bootstraps a [PostHog](https://posthog.com/) deployment on a [Kuberne
 
 ## Prerequisites
 
-- [Google Cloud Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/)
 - [Kubernetes](https://kubernetes.io/) 1.6+ with Beta APIs enabled
 - [Helm](https://helm.sh/) >= v3
 
+## Setting up Kuberentes Cluster
+
+<details>
+  <summary>
+    <b>Google Cloud</b>
+  </summary>
+<br />
+See [Google Cloud Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/)
+</details>
+
+<details>
+  <summary>
+    <b>AWS</b>
+  </summary>
+<br />
+TODO
+</details>
+
+<details>
+  <summary>
+    <b>Digital Ocean</b>
+  </summary>
+<br />
+Note that there is a [one click](todo: link) option to deploy Posthog on DigitalOcean
+</details>
+
 ## Installing the chart
 
-To install the chart with the release name `posthog`, run the following:
+To install the chart with the release name `posthog` in `posthog` namespace, run the following:
 
 ```console
 helm repo add posthog https://posthog.github.io/charts-clickhouse/
 helm repo update
-helm install -f values.yaml --timeout 20m posthog posthog/posthog
+helm install -f values.yaml --timeout 20m --create-namespace --namespace posthog posthog posthog/posthog
+```
+
+Where the `values.yaml` has the following minimum content:
+<details>
+  <summary>
+    <b>Google Cloud</b>
+  </summary>
+<br />
+
 ```
+cloud: "gcp"
+  nginx:
+    enabled: false
+certManager:
+  enabled: false
+```
+
+</details>
+
+<details>
+  <summary>
+    <b>AWS</b>
+  </summary>
+<br />
+
+```
+cloud: "aws"
+ingress:
+  hostname: <your-hostname>
+```
+</details>
+
+<details>
+  <summary>
+    <b>Other using nginx ingress controller</b>
+  </summary>
+<br />
+
+```
+ingress:
+  hostname: <your-hostname>
+```
+
+</details>
 
 The above command deploys PostHog on the Kubernetes cluster in the default configuration.
 
-You can find an example `values.yaml` file as well as an overview of the parameters that can be configured during installation under [configuration](#configuration).
+You can find an overview of the parameters that can be configured during installation under [configuration](#configuration).
 
-> **Tip**: List all releases using `helm list`
+### Setting up DNS
 
-### Setting up a static IP on Google Cloud
+<details>
+  <summary>
+    <b> Google Cloud prerequisite </b>
+  </summary>
+<br />
+
+Set up a static IP
 
 1. Open the Google Cloud Console
 1. Go to VPC Networks > [External IP addresses](https://console.cloud.google.com/networking/addresses/list)
 1. Add new static IP with the name `posthog`
 
-### Setting up DNS
+</details>
 
 After the chart has started, you can look up the external ip via the following command:
 
 ```bash
-kubectl get svc posthog
+kubectl get svc posthog --namespace posthog
 ```
 
 Create a `CNAME` record from your desired hostname to the external IP.
 
+### Setting up email
```

Comment:
Structure-wise this is awkwardly placed. This should be under the `configuration` section as everything else.

Reply:
Sure this is kind of optional, so happy to move.